docs/authentication.md

```diff
@@ -4,6 +4,8 @@ When using HTTPS, Helios client can present an X.509 client certificate to verif
 the user's identity. By presenting a digital signature to the Helios master, the
 client proves that it is who it claims to be.
 
+Note that Helios itself only contains functionality that lets the client present certificates. The Helios masters do not verify certificates themselves. The cluster administrator must add the missing pieces that validate certificates and act accordingly.
```

Comment:
But it doesn't mention that the master doesn't support HTTPS, so maybe we can leave something about that here?